spec/hosts/nodes/lsstcam-db01.ls.lsst.org_spec.rb

```diff
@@ -52,6 +57,32 @@
         it { expect(nm_keyfile['connection']['type']).to eq('ethernet') }
         it { expect(nm_keyfile['connection']['autoconnect']).to be_nil }
       end
+
+      context 'with enp129s0f1.2505' do
```

Comment:
Does -db01 need an LHN interface?

Reply:
I don't know what's going on here, I just made the test consistent with the existing hiera.
I note that profile::nm::connections in node/lsstcam-db01.ls.lsst.org.yaml is identical to that in site/ls/cluster/lsstcam-ccs.yaml, but with enp129s0f1.2505 and br2505 removed.
On the face of it, this does not make sense, given the merge strategy for this item.

Reply:
Maybe there needs to be a separate clean-up of this area?

Reply:
The live system already has an lhn interface, so it doesn't make sense to hold on merging this PR, but we should probably fix that.

Reply:
Jira issue to remove the lhn interfaces from hosts which should not have it: https://jira.lsstcorp.org/browse/IT-4387